projects/habitat_ovmm/scripts/summarize_metrics.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import json
+import os
+import sys
+from enum import IntEnum, auto
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+
+sys.path.append("projects/habitat_ovmm/")
+from utils.metrics_utils import get_stats_from_episode_metrics
```

Comment:
There is no `utils.metrics_utils.py` file

Reply:
Added